### DISCUSSION TITLE
Does MagicBell support SES SendTemplatedEmail function?

### DISCUSSION THREAD

**Discussion by gamer22026** [magicbell/community#109] [Questions] [answered]
We make use of SES templates to format email. Does MagicBell support the SendTemplatedEmail API in SES rather then just https://docs.aws.amazon.com/ses/latest/APIReference/API_SendEmail.html ?

**Comment by unamashana** [comment, marked as answer]
Answered here - https://github.com/orgs/magicbell-io/discussions/64#discussioncomment-6074470
